marinerescue-frontend/src/App.scss

```diff
@@ -35,7 +39,19 @@ button {
   align-content: center;
   width: 100%;
   height: 100%;
+}
+
+.container {
+  background-color: $sunshine;
+  color: $grape;
+  padding-left: 20px;
+  padding-right: 20px;
+  align-items: center;
+  border-radius: 30px;
+}
```

Comment:
In future projects, you should be careful about the site-wide styles you add. The `.container` style you added is a nice general tool that can be used in many layouts. Are you sure that every single container in our project should have an orange background and purple font?